bam2depth.c

```diff
@@ -365,6 +365,21 @@ int main_depth(int argc, char *argv[])
         }
     }
 
+    if (!ret && pos < 0 && all > 1 && !reg) {
```

Comment:
While this works fine, could it be merged with the "Handle terminating region" part above to avoid some duplication?  It should be possible to get it to produce the desired output with something like this just before the while loop:
```c
if (pos < 0 && last_tid < 0 && !reg) {
   last_tid = 0;
}
```

Reply:
Yes, it's a more elegant solution.